sql/common/V2.45.0__NWP_update_Crooked_River_Watershed_extents.sql

```diff
@@ -0,0 +1,18 @@
+-- Crooked River watershed NWP district
+-- xmin,ymax (top left), xmax ymax (top right), xmax ymin (bottom right), xmin ymin (bottom left), xmin ymax (top left again)
+
+--NWP - Crooked River
+-- x_min  -6825762.86388243
+-- x_max  -6100604.304031842
+-- y_min  8258151.748351492
+-- y_max  9081730.308212077
+--Rounded:
```

Comment:
I am guessing these were copied over and not updated.  That is fine.  As long as the values in the set statement are correct.  probably could delete them. Looking at the initial one though
('070204a3-66d9-471c-bd6e-ab59ea0858bb','crooked-river','Crooked River',ST_GeomFromText('POLYGON ((-2006000 2668000, -1854000 2668000, -1854000 2512000, -2006000 2512000, -2006000 2668000))',5070),'c18588b6-25ab-42c9-b31a-a33c585d0b49'),

I am wondering if what you have is in feet and not meters.

Reply:
the statement is correct.  just double check your min, max value to make sure that are in meters.

Reply:
K, i'll do that